__tests__/fixtures.js

```diff
@@ -23,11 +23,19 @@ const PAGES = {
   NO_ACCOUNTS: `${FIXTURES_CONST.BASE_URL}/no-accounts`,
   SETTINGS: `${FIXTURES_CONST.BASE_URL}/settings`,
   ADD_ACCOUNT_MNEMONIC: `${FIXTURES_CONST.BASE_URL}/add-account/mnemonic_phrase`,
-  SEND: `${FIXTURES_CONST.BASE_URL}/send`
+  ADD_ACCOUNT_VIEWONLY: `${FIXTURES_CONST.BASE_URL}/add-account/view_only`,
+  SEND: `${FIXTURES_CONST.BASE_URL}/send`,
+  ADD_ACCOUNT: `${FIXTURES_CONST.BASE_URL}/add-account`
 };
 
 const NETWORK_NAME_FIXTURE = 'Ropsten';
 
+const FIXTURE_ETHEREUM = 'Ethereum';
+
+const FIXTURE_VIEW_ONLY_ADDRESS = '0x82D69476357A03415E92B5780C89e5E9e972Ce75';
+
+const FIXTURE_VIEW_ONLY_TOKENS = ['Aragon', 'Basic Attention Token', 'Dai Stablecoin v2.0'];
```

Comment:
Prefer Asset symbol then name. The symbol seems to be more  stable then the name